config/_shared.json

```diff
@@ -256,7 +256,8 @@
 		"guided",
 		"domain-for-gravatar",
 		"plans-first",
-		"onboarding-affiliate"
+		"onboarding-affiliate",
+		"email-subscription"
```

Comment:
Note: These changes are meant to show the reskinned, white background signup flow styling instead of the older, outdated blue background styling